app/models/models_raid.py

```diff
@@ -0,0 +1,108 @@
+    address: Mapped[str] = mapped_column(String, nullable=False)
+    phone: Mapped[str] = mapped_column(String, nullable=False)
+    email: Mapped[str] = mapped_column(String, nullable=False)
+    bike_size: Mapped[Size | None] = mapped_column(Enum(Size), nullable=True, default=None)
+    t_shirt_size: Mapped[Size | None] = mapped_column(Enum(Size), nullable=True, default=None)
+    situation: Mapped[str | None] = mapped_column(String, nullable=True, default=None)
+    other_school: Mapped[str | None] = mapped_column(String, nullable=True, default=None)
+    company: Mapped[str | None] = mapped_column(String, nullable=True, default=None)
+    diet: Mapped[str | None] = mapped_column(String, nullable=True, default=None)
+    id_card_id: Mapped[str | None] = mapped_column(
+        ForeignKey("raid_document.id"), nullable=True, default=None
+    )
+    id_card: Mapped[Document] = relationship("Document", foreign_keys=[id_card_id])
+    medical_certificate_id: Mapped[str | None] = mapped_column(
+        ForeignKey("raid_document.id"), nullable=True, default=None
+    )
+    medical_certificate: Mapped[Document] = relationship(
+        "Document", foreign_keys=[medical_certificate_id]
+    )
+    security_file_id: Mapped[str | None] = mapped_column(
+        ForeignKey("raid_security_file.id"), nullable=True, default=None
+    )
+    security_file: Mapped[SecurityFile] = relationship(back_populates="participant")
+    student_card_id: Mapped[str | None] = mapped_column(
+        ForeignKey("raid_document.id"), nullable=True, default=None
+    )
+    student_card: Mapped[Document] = relationship(
+        "Document", foreign_keys=[student_card_id]
+    )
+    raid_rules_id: Mapped[str | None] = mapped_column(
+        ForeignKey("raid_document.id"), nullable=True, default=None
+    )
+    raid_rules: Mapped[Document] = relationship(
+        "Document", foreign_keys=[raid_rules_id]
+    )
+    attestation_on_honour: Mapped[bool] = mapped_column(Boolean, nullable=False, default=False)
+    validation_progress: Mapped[float] = mapped_column(Float, nullable=False, default=0.0)
+    payment: Mapped[bool] = mapped_column(Boolean, nullable=False, default=False)
+
+
+class Team(Base):
+    __tablename__ = "raid_team"
+    id: Mapped[str] = mapped_column(
+        String, primary_key=True, index=True, nullable=False
+    )
+    name: Mapped[str] = mapped_column(String, nullable=False)
+    number: Mapped[int] = mapped_column(Integer, nullable=False)
+    difficulty: Mapped[Difficulty] = mapped_column(Enum(Difficulty), nullable=True)
+    captain_id: Mapped[str] = mapped_column(
+        ForeignKey("raid_participant.id"), nullable=False
+    )
+    captain: Mapped[Participant] = relationship(
+        "Participant", foreign_keys=[captain_id]
+    )
+    second_id: Mapped[str | None] = mapped_column(
+        ForeignKey("raid_participant.id"), nullable=True
+    )
+    second: Mapped[Participant] = relationship("Participant", foreign_keys=[second_id])
```

Comment:
I'm not completely sure what would happen for the relationship `second` if `second_id` is None. Would `second` be none in that case?

Reply:
I tried to make second nullable, but relationship does not take nullable as argument, and this does not throw any error